src/components/SimulationVuer.vue

```diff
@@ -75,6 +75,17 @@ const IdType = Object.freeze({
   RAW_COMBINE_ARCHIVE: 'raw_combine_archive',
 });
 
+function isWebProtocol(urlString) {
+  try {
+    const url = new URL(urlString);
+    // protocol property includes the colon, e.g., "https:"
```

Comment:
Have comments start with a capital letter and a final point (same below).

Reply:
[0c304a3](https://github.com/ABI-Software/simulationvuer/pull/94/commits/0c304a3c0aceb2f6891bb1e4709b8142081faac3)